@here/olp-sdk-dataservice-read/lib/cache/KeyValueCache.ts

```diff
@@ -18,40 +18,42 @@
  */
 
 /**
- * Class for cache expecting a key,value pair.
+ * Used to provide the `KeyValueCache` instance to the `ApiCacheRepository` instance.
```

Comment:
not just to the `ApiCacheRepository` instance. Later we'll have more instances for more caching items (configuration, statistics, etc...) It's a base in-memory caching instance and all *Repository  will use it for reading or caching information

Reply:
Fixed